recipes/usockets/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8.11)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} test_package.cpp)
```

Comment:
minor, if we can change test to .c would be nice

Reply:
Renamed but have no idea why test_package return code is 32. Checked in docker with 4.9, still 32.

Reply:
Turned back to cpp due to problems with linking with gcc 4.9